sdk/core/azure-core/src/test/java/com/azure/core/util/polling/PollingStrategyTests.java

```diff
@@ -258,6 +259,61 @@ public void operationLocationPollingStrategyFailsOnPoll() {
         assertEquals(1, activationCallCount[0]);
     }
 
+    @Test
+    public void operationLocationPollingStrategyRelativePath() {
+        int[] activationCallCount = new int[1];
+        activationCallCount[0] = 0;
+        String endpointUrl = "http://localhost";
+        String mockPollRelativePath = "/poll";
+        String finalResultAbsolutePath = endpointUrl + "/final";
```

Comment:
the case in Haoling's comment is from deviceupdate that the final resource URL is also relative

e.g.
```
"resourceLocation": "/deviceUpdate/sdkinstance/updates/providers/fabrikam/names/vacuum/versions/2022.609.510.21?api-version=2022-07-01-preview"
```

Reply:
@mssfang Any update, as code freeze is closing?

Reply:
Sorry for the late reply. Do you mean resourceLocation could be either an absolute or relative path?

Reply:
Yes, in the case of deviceupdate backend, we found it is also relative (the ref on earlier comment is the real one we get).